src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -338,68 +338,87 @@ def k8s_install_connector(client, name, resource_group, connector_name,
     # Ensure that the SPN exists
     principal_obj = _ensure_service_principal(service_principal, client_secret, subscription_id,
                                               dns_name_prefix, location, connector_name)
-    client_secret = principal_obj.get("client_secret")
-    service_principal = principal_obj.get("service_principal")
+    client_secret = principal_obj.get('client_secret')
+    service_principal = principal_obj.get('service_principal')
     # Get the TenantID
     profile = Profile()
     _, _, tenant_id = profile.get_login_credentials()
+    # Check if we want the linux connector
+    if os_type.lower() in ['linux', 'both']:
+        _helm_install_aci_connector(url_chart, connector_name, service_principal, client_secret,
+                                    subscription_id, tenant_id, rgkaci.name, location,
+                                    node_prefix + '-linux', 'Linux')
+
     # Check if we want the windows connector
     if os_type.lower() in ['windows', 'both']:
-        # The current connector will deploy two connectors, one for Windows and another one for Linux
-        image_tag = 'canary'
-    logger.warning('Deploying the aci-connector using Helm')
+        _helm_install_aci_connector(url_chart, connector_name, service_principal, client_secret,
+                                    subscription_id, tenant_id, rgkaci.name, location,
+                                    node_prefix + '-win', 'Windows')
+
+def _helm_install_aci_connector(url_chart, connector_name, service_principal, client_secret,
+                                  subscription_id, tenant_id, aciResourceGroup, aciRegion,
+                                  nodeName, nodeOsType):
+    image_tag = 'latest'
+    node_taint = 'azure.com/aci'
+    helm_release_name = connector_name.lower() + "-" + nodeOsType.lower()
+    logger.warning("Deploying the ACI connector for '" + nodeOsType + "' using Helm")
```

Comment:
Change to `logger.warning("Deploying the ACI connector for '%s' using Helm", nodeOsType)`.
Also, can you make `nodeOsType` underscore cased?